src/cogent3/__init__.py

```diff
@@ -137,16 +137,22 @@ def make_unaligned_seqs(
     source
         origins of this data, defaults to 'unknown'. Converted to a string
         and added to info["source"].
+    new_type
```

Comment:
we need `new_type` arguments for all pertinent top level functions, e.g. `cogent3.get_code()`, `cogent3.get_moltype()` and `cogent3.make_seq()` etc.. Make those the next issue